src/App.css

```diff
@@ -0,0 +1,7 @@
+@import 'https://fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap';
+@import 'https://fonts.googleapis.com/css2?family=Marck+Script&display=swap';
+@import 'https://zero-ui.netlify.app/docs/main.css';
```

Comment:
Maybe adding these things as html links will be beneficial for [performance](https://www.geeksforgeeks.org/what-is-the-best-way-to-include-css-file-why-use-import/)

Reply:
Implemented.